docs/CMakeLists.txt

```diff
@@ -213,6 +213,10 @@ function(build_docs)
     set_target_properties(doxygen_xml sphinx_docs
                           PROPERTIES FOLDER docs)
 
+    install(FILES ${SPHINX_OUTPUT}
+            DESTINATION openvino_docs
+            COMPONENT openvino_docs
```

Comment:
why? we don't provide docs in packages

CC @slyubimt

Reply:
@ntyukaev Can you justify addition of install instructions? CMake install is used to define product content and install directory layout for our distributions.

Reply:
@ilya-lavrenov  @slyubimt  This is needed for CI. @ilya-lavrenov @azhogov  Can you have a look at my latest update? I've added "EXCLUDE_FROM_ALL" so it shouldn't be in the package.